### ISSUE TITLE
14_deep_computer_vision_with_cnns 파일에 괄호가 누락이 있는것 같습니다.

### ISSUE THREAD

**Issue by ziwon-seo** [rickiepark/handson-ml2#1]
Tacking Fashion Mnist with a CNN
부분에 

`In[28] : 
model.compile(loss="sparse_categorical_crossentropy", optimizer="nadam", metrics=["accuracy"])

> history = model.fit(X_train, y_train, epochs=10, validation_data=(X_valid, y_valid)
>>>끝에 괄호가 하나 빠진것 같습니다.

score = model.evaluate(X_test, y_test)
X_new = X_test[:10] # pretend we have new images
y_pred = model.predict(X_new)`

**Comment by rickiepark** [comment]
앗... 그러네요. ㅠ.ㅠ
아마도 제가 노트북을 실행해 보면서 키보드를 잘못 두드린 것 같네요.
알려 주셔서 감사합니다. :)

**Comment by ziwon-seo** [comment]
좋은책 번역해주셔서 정말 감사합니다.
